resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -45,6 +46,7 @@
     private int ringBufferSizeInHalfOpenState = DEFAULT_RING_BUFFER_SIZE_IN_HALF_OPEN_STATE;
     private int ringBufferSizeInClosedState = DEFAULT_RING_BUFFER_SIZE_IN_CLOSED_STATE;
     private Duration waitDurationInOpenState = Duration.ofSeconds(DEFAULT_WAIT_DURATION_IN_OPEN_STATE);
+    private IntervalFunction waitIntervalFunctionInOpenState = null;
```

Comment:
I think we should not initialize to null, is there a default value we can use?

Reply:
Actually, we expect it to be null most of the time - if it is null we use the wait duration instead. See CircuitBreakerStateMachine.java:491